docs/tutorial/sparql-llm.md

```diff
@@ -0,0 +1,428 @@
+# Generating SPARQL queries using LLMs
+
+This tutorial will teach you how to create SPARQL queries by prompting a large language model (LLM). You may use whichever system you prefer, such as [ChatGPT](https://chatgpt.com/), [Gemini](https://gemini.google.com/), [Claude](https://www.anthropic.com/claude-code), etc. You are responsible for obtaining access to one of these tools prior to the session.
+
+
+## Preparation
+This tutorial uses LLMs, such as [ChatGPT](https://chatgpt.com/), [Gemini](https://gemini.google.com/), [Claude](https://www.anthropic.com/claude-code), etc. You are responsible for obtaining access and any to one of these tools prior to the session.
+
+The resulting SPARQL queries can be tested using https://yasgui.triply.cc/# (endpoint https://ubergraph.apps.renci.org/sparql) or using [ROBOT query](https://robot.obolibrary.org/query). You are responsible for obtaining access to yasgui and/or installing ROBOT. 
+
+
+## Prompting Best Practices
```

Comment:
The best practices come too early here. Being someone that is doing the tutorial, I want to see a step by step tutorial, with a discussion of best practices later on in the right context.

Reply:
thanks, moved to the end